src/pages/individual/Individual.jsx

```diff
@@ -46,30 +50,56 @@ export default function Individual() {
   const history = useHistory();
   const fieldSchemas = useIndividualFieldSchemas();
 
-  function refreshIndividualData() {
-    const queryKey = getIndividualQueryKey(id);
-    queryClient.invalidateQueries(queryKey);
-  }
+  const refreshIndividualData = useCallback(
+    () => {
+      const queryKey = getIndividualQueryKey(id);
+      queryClient.invalidateQueries(queryKey);
+    },
+    [queryClient],
```

Comment:
When would the queryClient change?

Reply:
I don't think it ever would. We set the queryClient instance once in App.
However, according to the [React documentation](https://reactjs.org/docs/hooks-reference.html#usecallback):
> The array of dependencies is not passed as arguments to the callback. Conceptually, though, that’s what they represent: every value referenced inside the callback should also appear in the dependencies array. In the future, a sufficiently advanced compiler could create this array automatically.
>
> We recommend using the exhaustive-deps rule as part of our eslint-plugin-react-hooks package. It warns when dependencies are specified incorrectly and suggests a fix.

When I set up the exhaustive-deps rule, and removed queryClient from the array, it warned:
```
/codex-frontend/src/pages/individual/individual.jsx
  58:5  warning  React Hook useCallback has missing dependencies: 'id' and 'queryClient'. Either include them or remove the dependency array  react-hooks/exhaustive-deps
 ``` 
**TODO**: Add `id` to the dependency array too.